pkgs/applications/video/lightworks/default.nix

```
@@ -76,13 +102,12 @@ let
     dontPatchELF = true;
   };
 
+in
 # Lightworks expects some files in /usr/share/lightworks
-in buildFHSEnv {
-  name = lightworks.name;
+buildFHSEnv {
+  inherit (lightworks) name;
```

```suggestion
  inherit (lightworks) pname version;
```

This will likely change the executable name from `lightworks-2023.2` to `lightworks`

---

Changed.